robottelo/api/utils.py

```diff
@@ -771,6 +771,35 @@ def wait_for_syncplan_tasks(repo_backend_id=None, timeout=10, repo_name=None):
         time.sleep(2)
 
 
+def wait_for_errata_applicability_task(host_id, search_rate=1, max_tries=10, poll_rate=None,
```

Comment:
do you realize that this will not be very useful in scenarios with 2 consecutive errata applicability triggers?
as the second check would match the first task and immediately return it.
Perhaps accepting a datetime parameter e.g. `started_after` or something that would allow user to specify the target task as much as possible could help.
What do you think?

Reply:
Yep, rewrote it partly for this reason. Please review again.